### ISSUE TITLE
Not Deleting Offline Stream Annoucements

### ISSUE THREAD

**Issue by ghost** [dawgeth/CouchBot#4]
![2017-04-11 2](https://cloud.githubusercontent.com/assets/27312957/24933271/580f0ef4-1eca-11e7-8d89-6ba303e12725.png)
Hello,

I am not sure if this is an error on my part just the bot acting up, but I turned on the Auto Delete when streamers go offline but yet the bots not deleting anything?

Hopefully you guys might know something

I have attached a screenshot of my current config with Couchbot

**Comment by dawgeth** [comment]
@KristoffTheAmzg - Auto Delete is only functional for Beam at the moment. I need to get this fixed. Making this a priority. Will update this when it's completed.

**Comment by dawgeth** [comment]
Not going to fix this feature.

**Comment by StrikerMan780** [comment]
Why not? My stream channel is getting polluted with old streams, and it's quite aggravating.

**Comment by dawgeth** [comment]
Think this got closed on accident 4 months back. This feature was implemented - other platforms delete offline streams. However, had a recent report that they don't.

Also adding a task to "Do Nothing" when streams go offline as well.

Re-opening this to track this work.

**Comment by StrikerMan780** [comment]
Alright, cool. Next time it happens I'll send a few screenshots, and whether or not it's consistent, to see if maybe there's some kind of pattern.

**Comment by dawgeth** [comment]
If Constellation is down (or CouchBot has disconnected from it) these won't get deleted. I need to write a cleanup task to check all "orphaned" streams - I'm going to write a new task to do this. As for now, I have confirmed the delete task is functional, I rewrote a portion of Mixer code last night and this seems to have cleaned that up. However, the cleanup task will be a new feature - so I'm adding this there.
